README.md

```diff
@@ -34,6 +34,7 @@ startupTasks:
     - Ivan
 - type: pull
   dataSource: persons
+  loadPreviousState: false
```

Comment:
I think `startFromLastResult` or `startFromLastReadRow` would be a more clear name for the parameter. Because the user doesn't know what is the state in pull task

Reply:
I would like to use load as analogy load from any storage and th2 has only one storage.
Maybe `loadLastReadRow` ?

Reply:
From my perspective, it is unclear what it _loads_ this last row for. What does it do with it? But you can go with the name you like I am fine with whatever choice you make